CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Unreleased
 
-- Fix: Add support to show-hide-content.js for form fields with more varied characters: ([PR 403](https://github.com/alphagov/govuk_frontend_toolkit/pull/403))
+- Breaking change: Add support for GA Universal in stageprompt.js
```

Comment:
Can we explain why this is a breaking change, and what users need to do?

Reply:
Good shout will update